jbpm-console-ng-showcase/src/main/java/org/jbpm/console/ng/server/impl/DashbuilderBootstrap.java

```diff
@@ -106,10 +111,10 @@ protected void registerDataSetDefinitions() {
                 .uuid(HUMAN_TASKS_WITH_USER_DATASET)
                 .name("Human tasks and users")
                 .dataSource(JBPM_DATASOURCE)
-                .dbSQL("select  t.activationTime, t.actualOwner, t.createdBy, "
```

Comment:
No, I was referring to this one https://github.com/droolsjbpm/jbpm-console-ng/commit/f22fbeed401eb8180573c93654c0ffd5bcd19399